pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1144,6 +1161,169 @@ func TestPostFilterPlugins(t *testing.T) {
 	}
 }
 
+func TestFilterPluginsWithNominatedPods(t *testing.T) {
+	podNominator := internalqueue.NewPodNominator()
+	tests := []struct {
+		name             string
+		preFilterPlugins []*TestPlugin
+		filterPlugins    []*TestPlugin
```

Comment:
why are those defined as lists if non of the tests define more than one plugin of each?

Reply:
updated.